### PR TITLE
Notify users about no GPU information supplied when CUDA is available

### PR DESCRIPTION
**Pull request by fridex**
## Description

Related: https://github.com/thoth-station/adviser/pull/2236/

### CODE REVIEW

**Comment by fridex** [comment]
/assign @mayaCostantini 

Ready to be reviewed.

File: prescriptions/_generic/cuda_no_gpu.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        hardware:
+        - gpu_models: [null]
+    run:
+      stack_info:
+      - type: WARNING
+        message: >-
+          A CUDA is available but no GPU information provided, adjust GPU
+          configuration option to have recommendations suited for hardware
+          available
+        link: no_gpu
```
Comment (mayaCostantini) [inline prescriptions/_generic/cuda_no_gpu.yaml:18]: Should we write 

```suggestion
        link: cuda_no_gpu
```

to keep a consistency with [gpu_no_cuda.yaml](https://github.com/thoth-station/prescriptions/blob/master/prescriptions/_generic/gpu_no_cuda.yaml) ?

**Review by mayaCostantini — COMMENTED** [review]
Just one doubt otherwise lgtm :100: :+1:

File: prescriptions/_generic/cuda_no_gpu.yaml (hunk 1 above)
Comment (fridex) [reply prescriptions/_generic/cuda_no_gpu.yaml:18]: This one is referring to the following justification: https://thoth-station.ninja/j/no_gpu

Comment (mayaCostantini) [reply prescriptions/_generic/cuda_no_gpu.yaml:18]: I see, thanks!

**Review by mayaCostantini — APPROVED** [review]

**Pull request merged**